action/protocol/producer/producerpb/producer.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+// To compile the proto, run:
+//      protoc --go_out=plugins=grpc:. *.proto
+syntax = "proto3";
+package producerpb;
+
+message Admin {
+    bytes admin = 1;
+    bytes blockReward = 2;
+    bytes epochReward = 3;
+}
+
+message Fund {
+    bytes totalBalance = 1;
+    bytes availableBalance = 2;
+}
+
+message RewardHistory {
+}
+
+message Account {
+    bytes balance = 1;
+}
+
+message Beneficiary {
```

Comment:
https://github.com/iotexproject/IOTX-voting-contracts/pull/4 added the beneficiary on the contract side, guess we could remove it form here now.

Reply:
ack